_posts/2017-03-20-Rails-5-1-rc1.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'Rails 5.1.0.rc1: Loving JavaScript, System Tests, Encrypted Secrets, and more'
+category: releases
+author: rafaelfranca
+published: true
+date: 2017-03-20 15:00 -04:00
+---
+
+We're happy to announce Rails 5.1.0.rc1 has been released. With the help of the community we polished the Rails 5.1
+release with more than [380 commits](https://github.com/rails/rails/compare/v5.1.0.beta1...5.1.0.rc1).
```

Comment:
@rafaelfranca this URL is missing a `v` as reported by @javan -- should be https://github.com/rails/rails/compare/v5.1.0.beta1...v5.1.0.rc1 instead